manifests/params.pp

```diff
@@ -22,12 +22,12 @@
       }
     }
     'Debian': {
-      $default_su_user = versioncmp($facts['operatingsystemmajrelease'], '14.00') ? {
-        1        => 'root',
+      $default_su_user = $facts['operatingsystem'] ? {
```

Comment:
This now fails on ubuntu 14.04, right? In our code versioncmp() returns 0 on 14.04